soccer/gameplay/single_robot_composite_behavior.py

```diff
@@ -10,6 +10,24 @@
 # Note: The method-resolution-order in python dictates that as we have it listed below, SingleRobotBehavior methods take precedence over CompositeBehavior methods
 class SingleRobotCompositeBehavior(single_robot_behavior.SingleRobotBehavior,
                                    composite_behavior.CompositeBehavior):
+    #If the behavior should not be restarted when it is assigned to a different robot, then restart_behavior_on_switch_robot must be false
+    def __init__(self,
+                 continuous=False,
+                 restart_behavior_on_switch_robot=True):
```

Comment:
eeeeh can this be renamed? Maybe `restart_on_switch` or maybe `autorestart`?

This is /really/ long. I would also reccomend adding documentation [see this guide](https://robojackets.github.io/robocup-software/md_doc_Documentation.html) for this method too.